src/main/java/net/liquidpineapple/pang/objects/DropRandomizer.java

```diff
@@ -27,37 +62,17 @@ public DropRandomizer(){
      * 8% 1 of 4 colored Gems 1500p
      * 2% A heart that gives a life and 1p
      *
-     * @param startX
-     * @param startY
+     * @param startX Start position X
+     * @param startY Start position Y
      */
     public void rollRandomdrop(double startX, double startY) {
-        int roll = randomizer.nextInt(100);
-        Drop randomDrop  = null;
-        if(0 <= roll && roll < 30){
-             randomDrop  = new Drop("/sprites/drops/coinBronze.png", startX, startY, 0, 1, 10);
-        }
-        else if(30 <= roll && roll <50 ){
-             randomDrop  = new Drop("/sprites/drops/coinSilver.png", startX, startY, 0, 1, 100);
-        }
-        else if(50 <= roll && roll <60 ){
-             randomDrop  = new Drop("/sprites/drops/coinGold.png", startX, startY, 0, 1, 1000);
-        }
-        else if(60 <= roll && roll <62 ){
-             randomDrop  = new Drop("/sprites/drops/gemBlue.png", startX, startY, 0, 1, 1500);
-        }
-        else if(62 <= roll && roll <64 ){
-             randomDrop  = new Drop("/sprites/drops/gemRed.png", startX, startY, 0, 1, 1500);
-        }
-        else if(64 <= roll && roll <66 ){
-             randomDrop  = new Drop("/sprites/drops/gemYellow.png", startX, startY, 0, 1, 1500);
-        }
-        else if(66 <= roll && roll <68 ) {
-             randomDrop  = new Drop("/sprites/drops/gemGreen.png", startX, startY, 0, 1, 1500);
-        }
-        else if(68 <= roll && roll <70 ) {
-             randomDrop  = new Drop("/sprites/drops/heart.png", startX, startY, 0, 1, 1, 1);
-        }
+
+        int length = randomList.size();
+        int roll = randomizer.nextInt(length);
+        Drop randomDrop = randomList.get(roll);
+
         if( randomDrop  != null){
```

Comment:
As said above, you'd want to check for a specific value. Another implementation could be done by doing something like this:
```java
private static final int AMOUNT_OF_DROPS = 0; //specify the desired amount of drops here
...
...
int roll = randomizer.nextInt(AMOUNT_OF_DROPS);

if(roll <randomList.size()) {
Drop randomDrop = randomList.get(roll);
...
}